packages/gui/src/data/properties.ts

```diff
@@ -504,6 +504,9 @@ export const properties: Record<string, PropertyData> = {
     type: 'keyword',
     keywords: ['left', 'right', 'none', 'inline-start', 'inline-end'],
   },
+  fontFamily: {
+    type: 'typography',
```

Comment:
I think font family is sort of a "one off" input type so it might make sense to just set type to `fontFamily`.

Reply:
makes sense